docs/_ext/link_fixer.py

```diff
@@ -0,0 +1,61 @@
+import re
+from typing import Any, Dict
+
+from docutils import nodes
+from sphinx import addnodes, application
+from sphinx.transforms import SphinxTransform
+
+
+class LinkTransformer(SphinxTransform):
+    """Transformer to make all internal links actually internal.
+
+    Transforms https://pydocstringformatter.readthedocs.io/en/latest/usage.html
+    into a link to docs/usage.rst.
+    """
+
+    # Set priority very low so that we run after everything else has been done
+    default_priority = 1000
+
+    def apply(self, **_: Dict[str, Any]) -> None:
+        """Apply the transformation."""
+        for node in self.document.traverse(nodes.reference):
+            if (
+                "refuri" in node
+                and node["refuri"].startswith(
+                    "https://pydocstringformatter.readthedocs.io"
```

Comment:
This should be a constant imo. Well we can search and replace it in this little script, so it's not that important.